Formula/apache-spark.rb

```diff
@@ -4,10 +4,16 @@ class ApacheSpark < Formula
   url "https://www.apache.org/dyn/closer.lua?path=spark/spark-2.1.0/spark-2.1.0-bin-hadoop2.7.tgz"
   version "2.1.0"
   sha256 "0834c775f38473f67cb122e0ec21074f800ced50c1ff1b9e37e222a0069dc5c7"
+  revision 1
   head "https://github.com/apache/spark.git"
 
   bottle :unneeded
 
+  patch do
+    url "https://github.com/apache/spark/pull/16429.patch"
```

Comment:
You should be able to use the commit URL instead now that the PR has been merged: https://github.com/apache/spark/commit/20e6280626fe243b170a2e7c5e018c67f3dac1db.patch